pkgs/top-level/all-packages.nix

```diff
@@ -10136,6 +10136,9 @@ with pkgs;
 
   importNpmLock = callPackages ../build-support/node/import-npm-lock { };
 
+  inherit (callPackages ../build-support/node/fetch-pnpm-deps { })
+    fetchPnpmDeps1 fetchPnpmDeps2;
```

Comment:
I was thinking about it, and it'd be very nice if `pnpm` versions were available here - this would invite people to play with them, and they would find out these `pnpm` versions exist, only if they would come up in `nix search`. Perhaps another approach would be to define `pnpm8` and `pnpm9` attributes, and each will have a `pnpm${v}.fetchDeps` attribute.

Reply:
While I agree with this, I think this would be a bit out of scope for this PR. Ideally we would promote pnpm to a standalone package (outside `nodePackages`) similar to `yarn` and provide some kind of `pnpmVersions` scope.